packages/imperative/src/config/cmd/auto-init/handlers/BaseAutoInitHandler.ts

```diff
@@ -153,18 +160,25 @@ export abstract class BaseAutoInitHandler implements ICommandHandler {
             if (params.arguments.forSure && params.arguments.forSure === true) {
                 // Clear layer, merge, generate schema, and save
                 ImperativeConfig.instance.config.api.layers.set(profileConfig);
-                ConfigSchema.updateSchema();
+                const schema = ConfigSchema.buildSchema(ImperativeConfig.instance.loadedConfig.profiles);
+                ImperativeConfig.instance.config.setSchema(schema);
```

Comment:
I made no conscience decision use build/setSchema.

In history, I see that you altered build/set to updateSchema 20 days ago.

When I later merged from next into auto-init, I probably just chose the implementation that was already in auto-init (which was build/set).

Should we use updateSchema here and in the test module? If so, I can change things back to updateSchema.

Reply:
The only real-world scenario that using updateSchema will cover is if the user happens to have a `user.config.json` with a schema pointing to something other than what the regular `config.json` point to. 
Probably a very uncommon edge case.
And in the case of the BaseAutoInitHandler, I'm not sure we need this extra verification since we are likely going to be creating the schema and config for the first time with these commands 😋

Reply:
Can't auto-init be run after the schema and config has been created, allowing the user to update settings? Do we need to use updateSchema for that scenario?